csharp/src/Google.Protobuf/ByteString.cs

```diff
@@ -23,6 +24,8 @@ namespace Google.Protobuf
     /// Immutable array of bytes.
     /// </summary>
     [SecuritySafeCritical]
+    [DebuggerDisplay("Length = {Length}")]
```

Comment:
Note: `ByteString` displays length rather than count to act more like array/span/memory.